release/src/main/scripts/build_release_candidate.sh

```diff
@@ -103,7 +103,7 @@ if [[ $confirmation = "y" ]]; then
 
   echo "-------------Staging Java Artifacts into Maven---------------"
   ./gradlew publish -PisRelease --no-parallel --no-daemon
-  echo "Please review all artifacts in https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.beam%22"
+  echo "Please review all artifacts in staging URL. e.g. https://repository.apache.org/content/repositories/orgapachebeam-1049/"
```

Comment:
This particular directory is just for this candidate, right?

Reply:
Change the id to generic NNNN. It was meant to be an example.